catalyst/core/callbacks/metrics.py

```diff
@@ -165,7 +165,9 @@ def __init__(
                 Must be either ``sum``, ``mean`` or ``weighted_sum``.
             multiplier (float): scale factor for the aggregated metric.
         """
-        super().__init__(CallbackOrder.Metric + 1)
+        super().__init__(
+            order=CallbackOrder.MetricAggregation, node=CallbackNode.All
```

Comment:
why node is not Master?

Reply:
because you need to measure metric on all nodes